### PR TITLE
Add PrePostProcessor.output()

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - Expose ppp.output().tensor().set_element_type()
 - Update samples and remove TODO

### Tickets:
 - *[124059]*

### CODE REVIEW

File: src/bindings/js/node/include/preprocess/output_info.hpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <napi.h>
+
+#include <openvino/core/preprocess/output_info.hpp>
+
+class OutputInfo : public Napi::ObjectWrap<OutputInfo> {
+    public:
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/include/preprocess/output_info.hpp:10]:
```suggestion
public:
```

File: src/bindings/js/node/src/preprocess/output_tensor_info.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        try {
+            auto type = js_to_cpp<ov::element::Type_t>(info, 0, {napi_string});
+            _tensor_info->set_element_type(type);
+        } catch (std::exception& e) {
+            reportError(info.Env(), e.what());
+        }
+    } else {
+        reportError(info.Env(), "Error in setElementType(). Wrong number of parameters.");
+    }
+    return info.This();
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/preprocess/output_tensor_info.cpp:43]:
```suggestion
    if (info.Length() != 1) {
        reportError(info.Env(), "Error in setElementType(). Wrong number of parameters.");
    }

    try {
        auto type = js_to_cpp<ov::element::Type_t>(info, 0, {napi_string});
        _tensor_info->set_element_type(type);
            
        return info.This();
    } catch (std::exception& e) {
        reportError(info.Env(), e.what());
    }
```

File: src/bindings/js/node/src/preprocess/pre_post_process_wrap.cpp [hunk 3]
```diff
@@ -54,6 +55,26 @@ Napi::Value PrePostProcessorWrap::input(const Napi::CallbackInfo& info) {
     return obj;
 }
 
+Napi::Value PrePostProcessorWrap::output(const Napi::CallbackInfo& info) {
+    if (info.Length() != 0 && info.Length() != 1) {
+        reportError(info.Env(), "Wrong number of parameters.");
+        return info.Env().Undefined();
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/preprocess/pre_post_process_wrap.cpp:61]: Why is there a return after `reportError` in one place, but not in another?

**Review by vishniakov-nikolai — APPROVED** [review]
Looks good! I left couple codestyle comments.

File: src/bindings/js/node/include/preprocess/output_info.hpp (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/node/include/preprocess/output_info.hpp:10]: 👍

File: src/bindings/js/node/src/preprocess/pre_post_process_wrap.cpp (hunk 3 above)
Comment (almilosz) [reply src/bindings/js/node/src/preprocess/pre_post_process_wrap.cpp:61]: discussed offline

**Pull request merged**